### ISSUE TITLE
Docker image not synced last version 

### ISSUE THREAD

**Issue by a-legrand** [airbytehq/airbyte#26206]
### Octavia CLI Version

0.42.1

### Revelant information

Hello!

The latest Octavia version mentioned in doc and in the install.sh script is `0.44.4`
But this version is not available on Docker Hub, last available is `0.42.1`
Can you update it so we can align Airbyte server & cli versions and avoid inconsistencies? I am facing an issue with a connection parameter I think coming from this difference.

thank you!

### Relevant log output

```shell
docker pull airbyte/octavia-cli:0.44.4
Error response from daemon: manifest for airbyte/octavia-cli:0.44.4 not found: manifest unknown: manifest unknown
```

**Comment by rileybrook** [comment]
The Octavia image will be updated to 0.44.4 today or early next week 🙂

**Comment by terencecho** [comment]
Thank you for bringing this up! We've released the `0.44.4` image which can be found on [docker hub](https://hub.docker.com/layers/airbyte/octavia-cli/0.44.4/images/sha256-2dbd940e2ade04139f3a39d774080d54c24668c2d3c22ed3f1afe7112085c98b?context=explore) or by running `docker pull airbyte/octavia-cli:0.44.4`.

We've also added an item in our backlog to make sure that the Octavia CLI releases through the same process as the rest of the OSS release.
